src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,75 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final String PACKAGE = "mate.jdbc";
+    private static final Injector injector = Injector.getInstance(PACKAGE);
+    // i've taken out this list of main because i have a problem
+    // with declaration length usage (maven)
+    private static List<Driver> catDrivers = new ArrayList<>();
+    private static List<Driver> teslaDrivers = new ArrayList<>();
+    private static List<Driver> banderoMachineDrivers = new ArrayList<>();
```

Comment:
move these lists to main, describe your problem in your private channel or questions channel with screens to resolve it, it's not ok to make such variables global